### PR TITLE
Char counter function

### PR DESCRIPTION
**Pull request by DarkJediDJ**
## Checklist
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No error nor warning in the console.
## New functionality:
* Added Tests for function Reverse
* Added function Reverse
* README modified 
* Added Tests for function Calculator
* Added function Calculator

### CODE REVIEW

**Review by Hargeon — CHANGES_REQUESTED** [review]

**Review by Hargeon — CHANGES_REQUESTED** [review]

**Comment by DarkJediDJ** [comment]
> Read about semver and create a release.

Did it

**Review by Hargeon — APPROVED** [review]

File: reverse_test.go [hunk 1]
```diff
@@ -28,3 +28,27 @@ func TestReverse(t *testing.T) {
...
 }
+
+func TestCalculator(t *testing.T) {
+	testCalculatorCases := []struct {
+		name string
+		text string
+		want int
+	}{
+		{"Simple string", "Example", 7},
+		{"Russian string", "Россия!!!", 9},
```
Comment (gordonBusyman) [inline reverse_test.go:39]: 🙃

File: reverse_test.go [hunk 2]
```diff
@@ -28,3 +28,27 @@ func TestReverse(t *testing.T) {
...
+		{"Chinese string", "你好, 世界。", 7},
+		{"Lots of emojis", "\U0001f60a \U0001f616 \U0001f644 \U0001f925 \U0001f634 \U0001f635 \U0001f480", 13},
+		{"One emoji", "\U0001f60a", 1},
+		{"Empty string", "", 0},
+		{"String with tabs and spaces", "  		", 4},
+		{"Very long string", "oy4iGAPpxkz1NfJd4wak3wAIZ4xfSUFOjlTpLoO1OEQd26vXaBCS23cSnCPPFSOBI0H44T", 70},
+	}
+	for _, tc := range testCalculatorCases {
+		t.Run(fmt.Sprintf("Couldnt count chars in string: %s in case : %s", tc.text, tc.name), func(t *testing.T) {
```
Comment (gordonBusyman) [inline reverse_test.go:48]: What does it print? 
Check the signature of t.Run `func (t *T) Run(name string, f func(t *T)) bool {}` - so what is the first param?

**Review by gordonBusyman — CHANGES_REQUESTED** [review]
Please change the test names in both cases

**Review by gordonBusyman — CHANGES_REQUESTED** [review]
- Rename the testcases back, calculator already defined in `TestCalculator`
- Please split the files: reverse in reverse, calculator in calculator

**Review by gordonBusyman — COMMENTED** [review]
Pls move `calculator` related tests to `calculator_test` and we are gtg

**Review by gordonBusyman — APPROVED** [review]

**Pull request merged**